sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -99,6 +100,18 @@
     static final String LOCAL_FILE_PATH = "src/test/resources/sample_files/Test/";
     static final String ENCODED_EMPTY_SPACE = "{\"source\":\"https://fakeuri.com/blank%20space\"}";
 
+    // Business Card fields
+    static final String CONTACT_NAMES = "ContactNames";
+    static final String JOB_TITLES = "JobTitles";
+    static final String DEPARTMENTS = "Departments";
+    static final String EMAILS = "Emails";
+    static final String WEBSITES = "Websites";
+    static final String MOBILE_PHONES = "MobilePhones";
+    static final String OTHER_PHONES = "OtherPhones";
+    static final String FAXES = "Faxes";
+    static final String ADDRESSES = "Addresses";
+    static final String COMPANY_NAMES = "CompanyNames";
```

Comment:
Can this be a list of fields and you loop over them to call `validateFieldValueTransforms` when testing?

Reply:
Not sure if I follow it. Those fields are static fields. Looping all of them isn't an extra cost?

Reply:
updated